ack/readers/mytarget/reader.py

```diff
@@ -168,7 +169,11 @@ def split_content_by_date(self, content: List[Dict[str, Any]]):
                 content_by_date.append(new_line)
         yield from content_by_date
 
-    @retry(wait=wait_exponential(multiplier=1, min=1, max=600), stop=stop_after_delay(600))
+    @retry(
+        wait=wait_exponential(multiplier=1, min=1, max=600),
```

Comment:
Looking at the time it took for each id, can you confirm that this 10 mins limit still makes sense ?

Reply:
Yes, it still makes sense